pkg/cluster/controllers/storage/minio.go

```diff
@@ -224,7 +224,7 @@ func (m *MinIOController) getMinIONamespacedName() types.NamespacedName {
 func (m *MinIOController) getMinIOSecretNamespacedName() types.NamespacedName {
 	return types.NamespacedName{
 		Namespace: m.HarborCluster.Namespace,
-		Name:      DefaultPrefix + m.HarborCluster.Name + "-" + DefaultCredsSecret,
+		Name:      m.HarborCluster.Spec.InClusterStorage.MinIOSpec.SecretRef,
```

Comment:
@soulseen 

Can we make this field be an optional item? When the user does not provide, we keep the original default behavior that is auto-generating a random secret.

Reply:
agree, and updated.